providers/elasticsearch/tests/unit/elasticsearch/log/test_es_task_handler.py

```diff
@@ -70,9 +76,10 @@ def get_ti(dag_id, task_id, logical_date, create_task_instance):
 class TestElasticsearchTaskHandler:
     DAG_ID = "dag_for_testing_es_task_handler"
     TASK_ID = "task_for_testing_es_log_handler"
+    RUN_ID = "run_for_testing_es_log_handler"
     LOGICAL_DATE = datetime(2016, 1, 1)
-    LOG_ID = f"{DAG_ID}-{TASK_ID}-2016-01-01T00:00:00+00:00-1"
-    JSON_LOG_ID = f"{DAG_ID}-{TASK_ID}-{ElasticsearchTaskHandler._clean_date(LOGICAL_DATE)}-1"
+    LOG_ID = f"{DAG_ID}-{TASK_ID}-{RUN_ID}--1-1"
```

Comment:
nit: How about using str format to show the LOG_ID for better readability?
```suggestion
    LOG_ID = f"{DAG_ID}-{TASK_ID}-{RUN_ID}-{MAP_INDEX}-{TRY_NUM}"
```

Reply:
Sure & revised.